docs/integrations/dbt.md

```diff
@@ -22,36 +25,43 @@ The target specified in your `profiles.yml` file will be used by default. The ta
-* dbt tests are considered [audits](/concepts/audits) in SQLMesh. SQLMesh tests are [unit tests](/concepts/tests), which test query logic before applying a plan.
-* SQLMesh's incremental models track which intervals have been filled and automatically detects and fills interval gaps. dbt does not support intervals and their recommended incremental logic is not compatible, requiring small tweaks to the models (don't worry dbt compatibility is maintained).
+* SQLMesh will detect and deploy new or modified seeds as part of running the `plan` command and applying changes - there is no separate seed command. Refer to [seed models](../concepts/models/seed_models.md) for more information.
+* The `plan` command dynamically creates environments, so environments do not need to be hardcoded into your `profiles.yml` file as targets. To get the most out of SQLMesh, point your dbt profile target at the production target, and let SQLMesh handle the rest for you.
+* The term "test" has a different meaning in SQLMesh and dbt: 
+    - dbt "tests" are [audits](../concepts/audits.md) in SQLMesh
+    - SQLMesh "tests" are [unit tests](../concepts/tests.md), which test query logic before applying a SQLMesh plan
+* SQLMesh's incremental models track which intervals have been filled and automatically detect and fill interval gaps. dbt does not support intervals, and their recommended incremental logic is not compatible with SQLMesh, requiring small tweaks to the models (don't worry - dbt can still use the models!).
 
 ## How to use SQLMesh incremental models within dbt
 
-SQLMesh's incremental models track uses true incremental models, which are capable of detecting and backfilling any missing intervals. dbt's incremental logic does not support intervals, and is not compatible with SQLMesh.
+SQLMesh's incremental models automatically detect and backfill any missing intervals. dbt's incremental logic does not support intervals, and is not compatible with SQLMesh.
 
 ### Mapping dbt incremental to SQLMesh incremental
-SQLMesh supports [idempotent](/concepts/glossary#idempotency) incremental loads through the use of merge (sqlmesh calls this `incremental_by_unique_key`) and insert-overwrite/delete+insert (sqlmesh calls this `incremental_by_time`) incremental strategies. Append is not currently supported and not recommended due to not being idempotent.
+SQLMesh supports two approaches to implement [idempotent](../concepts/glossary.md#idempotency) incremental loads: 
+- Using merge (with the sqlmesh [`incremental_by_unique_key` model kind](../concepts/models/model_kinds.md#incremental_by_unique_key)) 
+- Using insert-overwrite/delete+insert (with the sqlmesh [`incremental_by_time_range` model kind](../concepts/models/model_kinds.md#incremental_by_time_range))
 
+Incremenatal loading with append is not currently supported and is not recommended since it is not idempotent.
 
 
 #### Merge modifications
-
+TODO
```

Comment:
So the model changes are the same for merge or insert-overwrite/delete+insert. The only difference is that the latter also needs a time column config parameter. Can we rework this section so that we don't have different sections for each?